netdeploy/remote/nodeConfig.go

```diff
@@ -16,6 +16,8 @@
 
 package remote
 
+import "github.com/libp2p/go-libp2p/core/peer"
```

Comment:
Made the change to use string in this pacakge since peer.ID is a type alias for string and removed it's use in the netdeploy package as well by creating a p2p package local notion of peerID.